### PR TITLE
we can now decode a stringified boolean

### PR DESCRIPTION
**Pull request by JeremyLecoeur**
# Description

When the trials module create review, it passes "False" which is currently not recognized by the python code. It applies a conversion that transform any string to True.

## Github Issue

https://app.clickup.com/t/3e5m9t

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Comment by rlmv** [comment]
Can we get a test for this, for both the true and false cases? Creating a model with a Boolean property, and creating records of that model should be sufficient.

Does the encoder need to be updated to serialize Boolean values to strings?

**Comment by JeremyLecoeur** [comment]
Working on the test.

I don't think we need to do the encoder though.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,9 +6,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 
+## 3.1.1
+
+### Fixed
+- Stringified boolean are now correctly deserialized to the proper value
+
 ## 3.1.0
 
-###Added
+###A dded
```
Comment (rlmv) [inline CHANGELOG.md]: Typo

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,9 +6,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 Documentation at <https://developer.blackfynn.io/python/>
 
+## 3.1.1
+
+### Fixed
+- Stringified boolean are now correctly deserialized to the proper value
```
Comment (rlmv) [inline CHANGELOG.md]: Suggestion: what about "- Fixed a bug in the deserialization of boolean properties" ?

**Pull request merged**